src/esm_runscripts/filedicts.py

```diff
@@ -115,7 +120,7 @@ class SimulationFile(dict):
         >>> sim_file.cp_to_exp_tree()  # doctest: +SKIP
     """
 
-    def __init__(self, full_config: dict, attrs_address: dict):
+    def __init__(self, full_config: dict, attrs_address: dict, separator="."):
```

Comment:
Don't make this an init parameter please. Will we ever in any case use a different one?
```suggestion
    def __init__(self, full_config: dict, attrs_address: dict):
```